fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -722,7 +722,7 @@ KeyValueStoreMemory::KeyValueStoreMemory( IDiskQueue* log, UID id, int64_t memor
 
 IKeyValueStore* keyValueStoreMemory( std::string const& basename, UID logID, int64_t memoryLimit, std::string ext ) {
 	TraceEvent("KVSMemOpening", logID).detail("Basename", basename).detail("MemoryLimit", memoryLimit);
-	IDiskQueue *log = openDiskQueue( basename, ext, logID);
+	IDiskQueue *log = openDiskQueue( basename, ext, logID, DiskQueueVersion::V0 );
```

Comment:
I've validated in correctness that we'd be fine flipping this to V1 passes restarting tests, so it can be set to V1 for 6.2.